File: BOJ/C++/1159.cpp
#include <bits/stdc++.h>
using namespace std;

string s, ret;
int N, cnt[26];
int main() {
  cin >> N;
  for (int i = 0; i < N; i++) {
    cin >> s;
    cnt[s[0] - 97]++;
  }

  for (int i = 0; i < 26; i++) {
    if (cnt[i] >= 5) {
      ret += char(i + 97);
    }
  }

  if (ret.size() == 0) {
    cout << "PREDAJA";
  } else {
    cout << ret;
  }

  return 0;
}

File: BOJ/JavaScript/1159.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = fs.readFileSync(filePath).toString().trim().split("\n");

const a = Array.from({ length: 26 }).fill(0);
let ret = "";
input.shift();

input.forEach((v) => {
  if (v.length > 0) {
    a[v[0].charCodeAt() - "a".charCodeAt()]++;
  }
});

a.forEach((v, idx) => {
  if (v >= 5) {
    ret += String.fromCharCode(idx + "a".charCodeAt());
  }
});

console.log(ret.length === 0 ? "PREDAJA" : ret);
